ait/dsn/bin/ait_sle_bridge.py

```diff
@@ -65,16 +65,17 @@ def process_pdu(raf_mngr):
 
 
 if __name__ == '__main__':
-    raf_mngr = ait.dsn.sle.RAF(hostname='atb-ocio-sspsim.jpl.nasa.gov', port=5100,
+    #put in if statement for each IP address
+    raf_mngr = ait.dsn.sle.RAF(hostname='137.228.89.105', port=5214, #ip address subject to change
```

Comment:
If this hostname, port, and inst_id is project-specific information it should not be included in the PR.

Reply:
The original hostname (atb-ocio-sspsim.jpl.nasa.gov) is also project specific. Should I add a sle_hostname field in config.yaml?